server/repository/src/db_diesel/diesel_schema.rs

```diff
@@ -13,6 +14,8 @@ use super::{
 };
 
 allow_tables_to_appear_in_same_query!(
+    latest_asset_log,
+    asset,
```

Comment:
I would prefer to move the following to `asset.rs`:

```rust
allow_tables_to_appear_in_same_query!(
    latest_asset_log,
    asset,
    store,
);
```
(I think these are the three tables, right?)

This macro generates code for all combinations of its argument, i.e. there will be a lot of unnecessary code generated.

Also its clearer which tables we want to have in the same query.